module_intros/density.RDF-BinWidth.ipynb

```diff
@@ -27,6 +27,8 @@
    "outputs": [],
```

Comment:
<p>Let's remove the changes to this file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>